src/maestro/step.py

```diff
@@ -75,6 +81,8 @@ async def run(self, *args, context=None, step_index=None):
         if self.step_loop:
             prompt = await self.loop(prompt, step_index=step_index)
             output["prompt"] = prompt
+        print(f"🐝 Response from {self.step_name}: {output['prompt']}")
```

Comment:
I believe you meant to include the bee here, iiuc that's only for when the beeai agent prints output